### PR TITLE
Adds the "Henchman" Title option to the blueshield

### PR DESCRIPTION
**Pull request by KazooBard**
## About The Pull Request

One more title, referring to goons or henchmen. I think goon is more gender neutral but that word is broken now :(

## Why It's Good For The Game

I think that not everything has to be very serious. This is a MRP server, we can allow people to be an "Alright boss" follower of the cap's. A little sillyness does good in the long run.

thepeoplewanttogoodn

## Changelog

:cl:
add: The blueshield now can select "Henchman" as their title
/:cl:

### CODE REVIEW

**Comment by KeRSedChaplain** [comment]
Add goon anyways RECLAIM THE WORD

**Comment by xXPawnStarrXx** [comment]
This is baby grade; We need "Boss/Baws" (Captain), "The muscle" (Blueshield), "Right-hand man/woman"(HoP), "Lead enforcer" (HoS) and sec should be "goons"

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm [hunk 1]
```diff
@@ -93,6 +93,7 @@
 		"Blueshield",
 		"Command Bodyguard",
 		"Executive Protection Agent",
+		"Henchman",
```
Comment (Majkl-J) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm:96]: Put this in modular_zubbers/code/modules/alternative_job_titles/code/alt_job_titles.dm as

```
/datum/job/blueshield/New()
	alt_titles |= list(
		"Henchman",
		)
	. = ..()
```

**Review by Majkl-J — CHANGES_REQUESTED** [review]

**Comment by projectkepler-ru** [comment]
> This is baby grade; We need "Boss/Baws" (Captain), "The muscle" (Blueshield), "Right-hand man/woman"(HoP), "Lead enforcer" (HoS) and sec should be "goons"

cory
I'm already a demon
image

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm (hunk 1 above)
Comment (KazooBard) [reply modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm:96]: Changes done as you said, baws!

**Review by Majkl-J — APPROVED** [review]

**Pull request merged**